server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1523,7 +1525,18 @@ public Optional<SplitComputations> getSplitComputations() {
         lastSplitComputation = new SoftReference<>(newComputation);
       } catch (IOException e) {
         lastSplitComputation.clear();
-        log.error("Failed to compute split information from files " + e.getMessage());
+        if (e.getMessage().contains("File does not exist")) {
```

Comment:
Can we check the class type instead of the string message, which could be localized.

```suggestion
        if (e.getMessage().contains("File does not exist")) {
```

Reply:
Changed this to check the exception's class type. 

Part of this PR is to make sure that if the file set changes due to compaction, it's not reported as an error message.

But it should still throw an error message if a file does not exist and it's expected to belong to that tablet.